src/instagram.erl

```diff
@@ -0,0 +1,314 @@
+-module(instagram).
```

Comment:
Yeah, I agree, twitter is actually called `twitter_oauth` - I wanted to hide authorization details from client's naming, client doesn't have to know that (in case it changes in the future).
Which one do you prefer: with or without `_client`?